src/CBT.NuGet/Tasks/GenerateNugetProperties.cs

```diff
@@ -115,6 +118,7 @@ public bool Execute(string packageRestoreFile, string[] inputs, string propsFile
             }
             catch (Exception e)
             {
+                Log.LogError(e.ToString());
```

Comment:
I am on the fence on this one.  I really don't like legit build errors being swallowed by default.  The case that lead me to this was a missing file error that was just being silently swallowed by this trace.  My concern with not having this is making certain situations being really hard to debug.  But on the other hand I am not sure if there won't be situations where a failure isn't just noise.  But even then I am of the thought that if it is a failure it should be addressed or accounted for in some manner and not just silently never noticed.